rules/integrations/google_workspace/persistence_mfa_disabled_for_google_workspace_organization.toml

```diff
@@ -24,7 +24,48 @@ interval = "10m"
 language = "kuery"
 license = "Elastic License v2"
 name = "MFA Disabled for Google Workspace Organization"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating MFA Disabled for Google Workspace Organization
+
+Multi-factor authentication (MFA) is a process in which users are prompted during the sign-in process for an additional form of identification, such as a code on their cell phone or a fingerprint scan.
+
+If you only use a password to authenticate a user, it leaves an insecure vector for attack. If the password is weak or has been exposed elsewhere, an attacker could be using it to gain access. When you require a second form of authentication, security is increased because this additional factor isn't something that's easy for an attacker to obtain or duplicate.
```

Comment:
```suggestion
If you only use a password to authenticate a user, it leaves an insecure vector for attack. If the users's password is weak or has been exposed elsewhere, an attacker could be use it to gain access. Requiring a second form of authentication increases security because attackers cannot easily obtain or duplicate the additional authentication factor.
```